modules/op-about-stepactions.adoc

```diff
@@ -0,0 +1,81 @@
+// This module is included in the following assemblies:
+// * about/understanding-openshift-pipelines.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-stepactions_{context}"]
+= StepAction
+
+A step is a part of a task. If you define a step in a task, you cannot reference this step from another task.
+
+However, optionally, you can define a `StepAction` custom resource (CR). This CR contains the action that a step performs. You can reference a `StepAction` object from a step to create a step that performs the action. You can also use resolvers to reference a `StepAction` object that is available in another namespace or from an external source.
+
+The following examples shows a `StepAction` CR named `apply-manifests-action`. This action applies manifests from a source tree to your {OCP} environment.
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: StepAction
+metadata:
+  name: apply-manifests-action
+spec:
+  params:
+  - name: working_dir
+    description: The working directory where the source is located
+    type: string
+    default: "/workspace/source"
+  - name: manifest_dir
+    description: The directory in source that contains yaml manifests
+    type: string
+    default: "k8s"
+  results:
+  - name: output
+    description: The output of the oc apply command
+  image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+  env:
+  - name: WORKING_DIR
+    value: $(params.working_dir)
+  - name: MANIFEST_DIR
+    value: $(params.manifest_dir)
+  script: |
+      #!/usr/bin/env bash
+      cd $WORKING_DIR
```

Comment:
We could leverage field `workingDir` and set it to `$(params.working_dir)` instead of this+env

Reply:
@ppitonak I tried to implement this but I don't know the YAML syntax well enough. I would appreciate your help here.

Reply:
I have now attempted this.